lib/Kafka/Librd.pm

```diff
@@ -167,8 +167,8 @@ close down the consumer
 
     $topic = $kafka->topic($name, \%config)
 
-return a L</Kafka::Librd::Topic>topic object, that can be used to produce
-messages
+Return a L<topic|/Kafka::Librd::Topic> object, that can be used to produce
```

Comment:
```suggestion
Return a L<Kafka::Librd::Topic> object, that can be used to produce
```

Let's spell this out for the benefit of documentation rendered without the target link like perldoc command line tool.

Reply:
Haha sorry I forgot there is no `Kafaka::Librd::Topic` document.  Lets leave this as is for now.